css/styles.scss

```
@@ -493,6 +493,11 @@ input:required:valid {
    border: 1px solid rgb(211, 211, 211) !important;
 }
 
+input:not(.submit):not([type=submit]):not([type=reset]):not([type=checkbox]):not([type=radio]):not(.select2-search__field)[type=color] {
```

Should be sufficient
```suggestion
input[type=color] {
```

---

This does not work, another rule has precedence

---

(that's what I've tried first ;))

---

ok